src/include/version.h

```diff
@@ -0,0 +1,109 @@
+/*-
```

Comment:
(Just a general comment, not necessarily something that needs fixing here.)

Out of curiosity, do we not normally put include guards on header files? If not, what do we do to protect against double-including something?

Reply:
Not that I've seen.. I have noticed that all the functions in `src/include/*.h` files need to be inline though.
@etienneptl any ideas on this?

Reply:
Oh, that jogs my memory - these are all static, so each compilation unit gets its own copy. I'm not sure about the inline thing, our code is all C89 so that's probably getting ignored depending on what C standard we end up using. (From memory, inline was a C99 thing)

Reply:
Sorry @ajmorton I don't have info about this.